sdk/ml/azure-ai-ml/azure/ai/ml/constants/_component.py

```diff
@@ -54,9 +54,9 @@ class ExternalDataType(object):
 
 
 class DataTransferBuiltinComponentUri(object):
-    IMPORT_DATABASE = "azureml://registries/azureml-dev/components/import_data_database/versions/0.0.1"
-    IMPORT_FILE_SYSTEM = "azureml://registries/azureml-dev/components/import_data_file_system/versions/0.0.1"
-    EXPORT_DATABASE = "azureml://registries/azureml-dev/components/export_data_database/versions/0.0.1"
+    IMPORT_DATABASE = "azureml://registries/azureml-preview/components/import_data_database/versions/0.0.1"
```

Comment:
might need to update some recordings

Reply:
since the component is not ready in master and will submit fail, so I don't update recording for now. 
I will update recording if it can work in cannary region

Reply:
seems submit to canary is ok, but run failed seems need backend to build environment for import/export taskhttps://ml.azure.com/experiments/id/f8a6680f-18bc-4fed-ab21-1096212c2d57/runs/bright_moon_s5wzywwpxc?wsid=/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourceGroups/sdk/providers/Microsoft.MachineLearningServices/workspaces/sdk-canary&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

Reply:
Maybe we introduce an environment variable that can control which registry will we take? 
default to azureml-preview

Reply:
it seems azureml-preview can submit and run successfully in master:https://ml.azure.com/experiments/id/b09f2042-2709-4e88-b896-5ec327757b4e/runs/test_777016802875?wsid=/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourceGroups/pipelineux/providers/Microsoft.MachineLearningServices/workspaces/wse2etests_datatransferv2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

Do we still need this env?

Reply:
Then we don't need it anymore.